main.go

```diff
@@ -21,12 +21,16 @@ import (
 var configPath string
 var cpuProfile string
 var memProfile string
+var checkMode bool
 var globalLogger = zerolog.New(os.Stderr).With().Timestamp().Logger()
 
 func main() {
 	flag.StringVar(&cpuProfile, "cpuprofile", "", "write cpu profile to `file`")
 	flag.StringVar(&memProfile, "memprofile", "", "write memory profile to `file`")
 	flag.StringVar(&configPath, "config", "", "configuration file")
+	flag.StringVar(&configPath, "f", "", "configuration file")
+	flag.BoolVar(&checkMode, "check", false, "check mode : only check config files and exit")
+	flag.BoolVar(&checkMode, "c", false, "check mode : only check config files and exit")
```

Comment:
@fionera What do you think about having there only the short version of the parameters?

The rewrite is already a breaking change, so changing the parameter isn't that big of a deal.
Also having the same syntax as HAProxy makes it easier to remember...

But making the profiling options short, isn't necessary, since they are here for debugging and not for the run in normal operation. I would leave them long.

```suggestion
	flag.StringVar(&configPath, "f", "", "configuration file")
	flag.BoolVar(&checkMode, "c", false, "check mode : only check config files and exit")
```

---
Second consideration (or maybe other PR)
And add a version flag to print it from the binary, it gets it from the VCS at buildtime (since [Go 1.18](https://tip.golang.org/doc/go1.18#go-version))
```go
flag.BoolVar(&version, "v", false, "print version and exit")

// ...

if version {
    var buildInfo, _ = debug.ReadBuildInfo()
    fmt.Printf("Version: %s", buildInfo.Main.Version)
    os.Exit(0)
}
```